src/Parameters/CreateMeetingParameters.php

```diff
@@ -321,6 +336,26 @@ public function setAutoStartRecording($autoStartRecording)
     }
 
     /**
+     * @return bool
+     */
+    public function isAllowStartStopRecording()
```

Comment:
Can you please rename this method to allowStartStopRecording, it is sufficient and self-explanatory.